node-runner-cli/scripts/docker_install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+#sudo apt-get update
+#sudo apt-get install -y ca-certificates curl gnupg
+#
+#sudo install -m 0755 -d /etc/apt/keyrings
+#curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+#sudo chmod a+r /etc/apt/keyrings/docker.gpg
+#
+#echo \
+#  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+#  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+#  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+#
+#sudo apt-get update
+#
+#sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+curl -fsSL https://get.docker.com -o get-docker.sh
+sudo sh get-docker.sh
+docker run hello-world
```

Comment:
Do you need to run the container?

Reply:
I would rather run it to make sure docker was successfully installed